webapp/channels/src/utils/desktop_api.ts

```diff
@@ -166,8 +166,8 @@ class DesktopAppAPI {
         url: string,
     ) => {
         if (window.desktopAPI?.sendNotification) {
-            window.desktopAPI.sendNotification(title, body, channelId, teamId, url, silent, soundName);
-            return;
+            const result = await window.desktopAPI.sendNotification(title, body, channelId, teamId, url, silent, soundName);
```

Comment:
How is this going to behave if you're on an existing version of the desktop app that supports `sendNotification` and not the ack response? Is that different from when the app doesn't support the desktop API at all?

Reply:
You just get an `undefined` which is why we have the coalesce there. I've done this here since the Desktop API is responsible for compatibility.

Reply:
K. As long as `undefined` is no longer a valid return value for this, that should be fine then